web/src/components/share-buttons/share-buttons.css

```diff
@@ -7,6 +7,14 @@
     }
 }
 
+.XIcon path {
+    color: var(--white);
+    font-size: var(--font-size-xs);
+}
+.XIcon path:hover {
+    color: var(--blue);
```

Comment:
You need to change this from `color` to `fill`. Currently, hovering on the icon does not change the colour to blue. Using `fill` changes that